JPS_Ontology/ontology/ontomops/ontomops.csv

```diff
@@ -0,0 +1,57 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoMOPs,TBox,https://www.theworldavatar.com/kg/ontomops/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoMOPs,TBox,1.1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoMOPs,TBox,An ontology developed for representing Metal-Organic Polyhedra (MOPs).,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoMOPs,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+MolecularCage,Class,,,,,,A molecular cage is chemical species that has a dimensional chamber (i.e. cavity) surrounded by a molecular framework. It is a form of a polyatomic entity. ,https://www.theworldavatar.com/kg/ontomops/,
+CoordinationCage,Class,MolecularCage,IS-A,,,,A coordination cage is a molecular cage comprised of metal-based and organic building units.  A metal-organic cage (MOC) may be a synonym for coordination cage. ,https://www.theworldavatar.com/kg/ontomops/,
+MetalOrganicPolyhedron,Class,CoordinationCage,IS-A,,,,A metal-organic polyhedron (MOP) is a highly symmetrical coordination cage.,https://www.theworldavatar.com/kg/ontomops/,
+http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Species,Class,,,,,,OntoSpecies is a web ontology for chemical species. ,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl,
```

Comment:
`OntoSpecies:Species` doesn't seem to be mentioned anywhere else in the latest `OntoMOPs`, perhaps we can remove it?

Reply:
Agreed, removed.
Commit 00268447a9ee1888e18ee7ad9c83cdde92a3c587